### DISCUSSION TITLE
Riscaldamento con Tado x in casa su tre piani

### DISCUSSION THREAD

**Discussion by peterpalace** [indomus/forum#3085] [Nuove idee, proposte, progetti e guide]
Ciao! 
Vorrei il vostro aiuto per capire se quello che ho in mente è fattibile o se c'è altra soluzione per la gestione del riscaldamento in una casa di tre piani.
Vorrei prendere i prodotti della linea tado X.
Dal punto di vista idraulico  ci sono
Due impianti “idraulicamente autonomi” collegati a **due** circolatori idraulici, uno per il piano terra, e l’altro per il primo e secondo piano.
Vorrei gestire il riscaldamento dividendo la casa in **3 zone**: una parte del piano terra (monolocale autonomo), il resto del piano terra con il piano 1 e il piano 2.
È possibile gestire le valvole Tado in questo modo? Dovrei installare 3 termostati?
Inoltre,domanda forse più importante, essendo la casa molto grande e divisa su più piani, volevo mettere più "ripetitori" e non ho capito se installare più Tado bridge X risponda a questa domanda.
Grazie.

**Comment by 50494554524F** [comment]
tado non ha le zone che si possono impostare generalmente sulle caldaie... una zona sono le valvole associate ad un termostato tado

puoi usare un termostato per ogni piano.. se devi mettere un termostato per ogni stanza devi crearti le zone in ha

se metti il bridge nel piano di mezzo dovrebbe bastare, altrimenti ovvio un bridge per ogni piano

**Reply by KC-inDomus** [reply]
la nuova generazione X non è così, prevede soluzioni più versatili

**Reply by peterpalace** [reply]
Più versatile? In che senso?
Quindi posso mettere più bridge? Anche 3 bridge e due termostati? Io purtroppo ho la caldaia che si "accende" per tutto il piano 2 e 3 o solo per il primo. Quindi se non gestisce le zone devo optare per due termostati e poi controllare da app.
